presentation/src/main/java/com/pocs/presentation/view/user/anonymous/AnonymousCreateScreen.kt

```diff
@@ -73,7 +69,15 @@ fun AnonymousCreateScreen(
                 .verticalScroll(scrollState)
                 .fillMaxWidth()
         ) {
-            EditGroupLabel("필수")
+            Box(Modifier.height(8.dp))
+            Text(
+                stringResource(R.string.anonymous_guideline),
+                style = MaterialTheme.typography.bodyMedium.copy(
+                    color = MaterialTheme.colorScheme.onBackground.copy(alpha = 0.5f)
+                ),
```

Comment:
줄간격이 아직 좁게 보이네요. 아래와 같이 추가해주면 됩니다.
```suggestion
                ),
                 lineHeight = 24.sp
```